test/platform.test.ts

```diff
@@ -20,4 +22,223 @@ suite('Platform Tests', () => {
             done();
         });
     });
+
```

Comment:
Why are you using chai expect as opposed to assert which is already a part of the vscode testing framework?